app/assets/scripts/components/documents/document-download-menu.js

```diff
@@ -37,6 +40,67 @@ export default function DocumentDownloadMenu(props) {
   const ability = useContextualAbility();
   const canDownloadJournalPdf = ability.can('download-journal-pdf', atbd);
 
+  const handlePdfDownloadClick = React.useCallback(() => {
+    const { id, version, alias } = atbd;
+    const pdfUrl = `${apiUrl}/atbds/${id}/versions/${version}/pdf`;
+    const pdfFileName = `${alias}-v${version}.pdf`;
+    let retryCount = 0;
+    let timeoutRef;
+
+    async function fetchPdf(url) {
+      const toast = createProcessToast('Downloading PDF, please wait...');
+      const user = await Auth.currentAuthenticatedUser();
+      if (!user) {
+        toast.error('Failed to download PDF! (Not authenticated)');
+        return;
+      }
+
+      const {
+        signInUserSession: { idToken, accessToken }
+      } = user;
+      const authToken = `Bearer ${idToken.jwtToken}`;
+      const xAccessToken = accessToken.jwtToken;
+
+      try {
+        const response = await fetch(url, {
+          method: 'GET',
+          headers: {
+            Authorization: authToken,
+            'X-ACCESS-TOKEN': xAccessToken
+          }
+        });
+
+        if (response.headers['content-type'] === 'application/json') {
+          if (retryCount < 3) {
+            window.clearTimeout(timeoutRef);
+            setTimeout(() => {
+              fetchPdf(pdfUrl);
+            }, 3000);
+            ++retryCount;
+          } else {
+            toast.error('Failed to download PDF!');
+          }
+        } else if (response.headers['content-type'] === 'application/pdf') {
+          const pdfBlob = new Blob([response.data], {
+            type: 'application/pdf',
+            encoding: 'UTF-8'
+          });
+
```

Comment:
```suggestion
          const pdfBlob = await response.blob();
```
This seems to work better. Otherwise `response.data` comes out as `undefined`